jenkins/runlocal

```diff
@@ -1,29 +1,30 @@
 #!/bin/bash -e
 
-# Build the RPM and then build the containers, pushing only the CI image
-# to the registry, and then run the fuctional tests against the built
-# container image.
-
-export PB_CONTAINER_REG=images.paas.redhat.com
-export PB_ORG_NAME=pbench
```

Comment:
There are two places where `PB_CONTAINER_REG` is now referenced, in subordinate files, `container-build-ci.sh` and `run-pbench-in-a-can`, but these two files provide independent defaults for the value.  As long as those defaults stay in sync, we're OK, but I think it would be better for this file to provide the default so that it ensures that they use consistent values.

Reply:
Should be resolved now.

Reply:
It looks to me like the two subordinate files are still providing their own defaults (which is fine), and they are providing them in the same way (which is important, but, I think, fragile).

It seems to me that this file should _also_ provide a default definition so that, when the other scripts are invoked by this one, their defaults are _not_ used because a single, common value is provided by this script.  (Or, am I missing something?)

Reply:
First, while it is pretty slick to build images locally using a target repository name, preventing us from having to track the differences between `localhost` and the target repository, it proliferates the need for that target repository knowledge.

While I don't like that, it _seems_ like avoiding the use of `localhost` is still worth it.

However, we have references to `images.paas.redhat.com/pbench` in 5 different files in our tree, and the thought of keeping this as a 6th was more disturbing than the fragility you mention.

What I'd like to do is introduce an environment variable where the various points of use would fail if that variable is not provided.  The CI already fetches the value via a secret, so we'll just need developers to set that environment variable when they want to run outside the CI.

I'll propose something.

Reply:
Posted a separate commit which might be a solution, though I still need to verify all the changes work if you think this is a good approach.